README.md

```diff
@@ -34,26 +34,29 @@ And much more...
 
 Building
 --------
-Before even building, Alure requires the OpenAL soft library (libopenal.dll, 
-libopenal.a, libopenal.dll.a) to be available and on the path of your system.  Additional 
-optional dependencies are used for playback of files, which include:
 
-* ogg: libogg.dll.a  
-* vorbis: libvorbisfile.dll.a  
-* flac: libFLAC.dll.a  
-* opusfile: libopusfile.dll.a  
-* SndFile: libsndfile.dll.a  
-* mpg123: libmpg123.dll.a  
-* PhysFS: libphysfs.dll.a  
-* dumb: libdumb.dll.a  
+#### - Dependencies -
+Before even building, Alure requires the OpenAL development files installed, for example, through 
+Creative's OpenAL SDK (available from openal.org) or from OpenAL Soft. 
 
-Some of the optional files are required for building the examples provided.  You can disable building 
-these examples if you wish by setting the cmake variable `ALURE_BUILD_EXAMPLES` to `OFF` or un-checking it in the cmake gui
+Additional 
+dependencies are required for native playback of audio files:
```

Comment:
Looks like an errant newline here. Also, the dependencies are only needed to automatically support the formats they handle; a plain wav file decoder is built-in to automatically handle basic wav files, and app-provided custom decoders are allowed, so they aren't strictly required to handle audio files.

Reply:
I'm going to add this pretty much word for word, give me asecond.  also adding the other fix you added.